src/fides/data/sample_project/postgres_sample.sql

```diff
@@ -0,0 +1,199 @@
+-- Example Postgres schema matching the dataset in public/data/dataset/postgres_example_dataset.yml
```

Comment:
Rethinking this a bit... rather than make a copy, should we just mount the example postgres/mongo data directly from the project's docker/ folder? This necessarily have to stay in sync with any changes to the sample dataset files which we haven't copied, so could break subtly over time

Reply:
I thought about this...I think we should reuse as much as possible to eliminate that class of bug (subtle divergences we don't catch) so we should update the volumes mounts for both `postgres` and `mongo` test docker containers to use the files in `sample_project` instead of the ones in `docker/sample_data` or whatever it is

Reply:
basically, there should be a single sample data file for each datastore

Reply:
Yeah- in that case, I think we need to point everything to `docker/sample_data` as the source of truth which is used in pytest, test_env, sample_project, etc.

For things like config files (TOML, config.json, etc.) I think it's better to have different versions for each (pytest, test_env, sample_project, etc.)

Reply:
I'm updating all of the references I can find